website/docs/cloud-docs/api-docs/changelog.mdx

```diff
@@ -9,9 +9,44 @@ description: >-
 
 Keep track of changes to the API for HCP Terraform and Terraform Enterprise.
 
-## 2024-07-18
+## 2025-07-18
```

Comment:
@ker-an  or @netramali Just a reminder we'll need to update this changelog before publishing